src/components/namestable/nametable-section.component.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { DataRow } from "../../logic/schedule-logic/data-row";
+
+export interface NameTableCellOptions {
+  dataRow: DataRow[];
+}
+
+export function NameTableSection({ dataRow }: NameTableCellOptions): JSX.Element {
+  function getNames(): string[] {
+    return dataRow.map((a) => a.rowKey);
+  }
+
+  const data = getNames();
+
+  return (
+    <React.Fragment>
+      <table className="nametable">
+        <tbody>
+          {data.map((cellData) => {
+            return (
+              <tr className="nametableRow">
```

Comment:
Brakuje klucza na `tr`. 
I też jest problem z tym, że jest niepoprawne zagnieżdżenie elementów w DOM, przez co przeglądarki pokazują takie warningi:
![image](https://user-images.githubusercontent.com/21079319/100811586-40cfb100-343b-11eb-9ce3-a3dae295010c.png)

Nie wiem, szczerze mówiąc czy to źle, ale z uwagi na to, że je bardzo łatwo usunąć - usunąłem